build.ps1

```diff
@@ -1,4 +1,8 @@
-param([string]$buildtfm = 'all', [switch]$NoMsbuild)
+param(
+	[ValidateSet("all","net-x86","net-x64")]
```

Comment:
Shouldn't this set also contain `netframework` since this argument is used to build only the .NET Framework version of dnSpyEx?

Reply:
Ha very much so.  Sorry added that correction near the very end where my test case was changing the framework to netx86 and it built successfully.